operators/tool_tracking_postprocessor/python/tool_tracking_postprocessor.cpp

```diff
@@ -73,12 +73,11 @@ class PyToolTrackingPostprocessorOp : public ToolTrackingPostprocessorOp {
   // Define a constructor that fully initializes the object.
   PyToolTrackingPostprocessorOp(
       Fragment* fragment, const py::args& args, std::shared_ptr<Allocator> device_allocator,
-      std::shared_ptr<Allocator> host_allocator, float min_prob = 0.5f,
+      float min_prob = 0.5f,
       std::vector<std::vector<float>> overlay_img_colors = VIZ_TOOL_DEFAULT_COLORS,
       std::shared_ptr<holoscan::CudaStreamPool> cuda_stream_pool = nullptr,
       const std::string& name = "tool_tracking_postprocessor")
       : ToolTrackingPostprocessorOp(ArgList{Arg{"device_allocator", device_allocator},
-                                            Arg{"host_allocator", host_allocator},
```

Comment:
If this is removed, I think you likely also need to update the other variants of the endoscopy tool tracking app that are using this operator:

- `applications/endoscopy_tool_tracking_distributed`
- `applications/h264/h264_endoscopy_tool_tracking`

Reply:
Thanks, I forgot these.